force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -137,18 +195,18 @@ public with sharing class FlowControlLoopsCollections {
      * Example: returnHelloWorld() should return "Hello World!; Hello World!; Hello World!; "
      * @return The string "Hello World!; Hello World!; Hello World!; "
      */
-    public static String returnHelloWorld() {
-        // Initialize the result String
-
-        // Use a for loop to append 'Hello World!;' to the result string three times
-        /* 
-         for (){
-            
-         }
-        */
-
-        return null; // Replace null with the variable you used to store the result
-    }
+        public static String returnHelloWorld() {
```

Comment:
Check the spacing on the public, try to line it up with the beginning of the other methods for better readability and consistency